Source/WebCore/page/scrolling/ScrollingCoordinatorTypes.h

```diff
@@ -84,8 +84,8 @@ struct ScrollableAreaParameters {
     bool allowsHorizontalScrolling { false };
     bool allowsVerticalScrolling { false };
 
-    bool horizontalScrollbarHiddenByStyle { false };
-    bool verticalScrollbarHiddenByStyle { false };
+    NativeScrollbarVisibility horizontalScrollbarHidden { NativeScrollbarVisibility::Visible };
```

Comment:
I think you should now rename `horizontalScrollbarHidden` and `verticalScrollbarHidden` to match the sense of the enum.

Reply:
Renamed these and the below functions to horizontalNativeScrollbarVisibility and verticalNativeScrollbarVisibility respectively.